sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -236,6 +236,14 @@ primaryExpression
     | IF '('condition=expr ',' trueValue=expr (',' falseValue=expr)? ')'             #ifCase
     ;
 
+primaryExpressionOrSubquery
```

Comment:
maybe `parenthesizedPrimaryExpression`would be a better name, see https://github.com/crate/crate/pull/4736/files#diff-45df16323f5236963206e8b54dbb3756R220